src/rshim_pcie_lf.c

```diff
@@ -644,6 +648,19 @@ static int rshim_boot_fifo_write(struct pci_dev *pci_dev, int addr,
   return 0;
 }
 
+static uint32_t
```

Comment:
Would love to see a function comment here.

Reply:
done